packages/server/tests/server.spec.ts

```diff
@@ -302,6 +302,18 @@ describe('RPC Server', async function() {
 
     BaseTest.unsupportedJsonRpcMethodChecks(res);
   });
+
+  it('should execute "eth_blockNumber"', async function() {
+    const res = await this.testClient.post('/', {
```

Comment:
Please mock the `AxiosInstance (this.testClient)` this will allow you to provide a valid response from the MirrorNode
I provided examples in `mirrorNodeClient.ts`

Reply:
Actually is this even passing?
The mirrorNode call should be failing.
If you add a UT test in the `eth.spec.ts` I think you should be fine to leave this out for now since the response is dynamic.
Once we have integration tests we can have this covered here